src/PureCloud.Client/Apis/JourneyApi.cs

```diff
@@ -28,7 +28,7 @@ public async Task DeleteJourneyActionmapAsync(string actionMapId, CancellationTo
 
         var client = _httpClientFactory.CreateClient(PureCloudConstants.PureCloudClientName);
 
-        var response = await client.DeleteAsync($"api/v2/journey/actionmaps/{actionMapId}", cancellationToken);
+        var response = await client.DeleteAsync($"api/v2/journey/actionmaps/{Uri.EscapeDataString(actionMapId)}", cancellationToken);
```

Comment:
All delete methods should return Task<bool> not Task. Instead of calling the EnaureSuccessfulStausCode(), just retun response.IsSucceasfulStatusCode